src/Listener/Uuid1.php

```diff
@@ -14,14 +14,15 @@ final class Uuid1
     public function __construct(
         private string $field = 'uuid',
         private Hexadecimal|int|string|null $node = null,
-        private ?int $clockSeq = null
+        private ?int $clockSeq = null,
+        private bool $generate = true
```

Comment:
What about `nullable` field instead of `generate`? 
Nullable field will be created as `nullable` in DB and it will mean disable UUID autogeneration

Reply:
Renamed